taxonomies/allergens.txt

```diff
@@ -30,7 +30,8 @@ sk:glutén, lepok, obyčajná, raž, jačmeň, ovos, špalda, cirok
 sv:gluten, vete, råg, korn, havre, spelt, kamut, vetemjöl, rågmjöl, kornmjöl, havremjöl, speltmjöl, kamutmjöl, vetekli, rågkli, kornkli, havrekli, speltkli, kamutkli, fullkornsvete, fullkornsråg, fullkornskorn, fullkornshavre, fullkornsspelt, fullkornskamut, fullkornsvetemjöl, fullkornsrågmjöl, fullkornshavremjöl, fullkornsveteflingor, fullkornsrågflingor, fullkornshavreflingor, fullkornsspeltflingor, fullkornskamutflingor, malt, vetemalt, vetemaltextrakt, rågmalt, rågmaltextrakt, kornmalt, kornmaltextrakt, havremalt, havremaltextrakt, speltmalt, speltmaltextrakt, kamutmalt, kamutmaltextrakt, vetegryn, råggryn, korngryn, havregryn, speltgryn, kamutgryn, vetegroddar, råggroddar, korngroddar, havregroddar, speltgroddar, kamutgroddar
 th:กลูเต็น,ข้าวสาลี,ข้าวไร,บาร์เลย์
 zh:麸质
-wikidata:en:Q4413030, Q2249305
+# wikidata:en:Q4413030, Q2249305
```

Comment:
```suggestion
```
I would remove this completely.

Reply:
I was on the fence about removing it, but decided to leave it as they are related to the allergy. I'll remove it.